src/snowflake/connector/arrow_context.py

```diff
@@ -88,12 +88,30 @@ def TIMESTAMP_NTZ_to_python(self, microseconds):
     def TIMESTAMP_NTZ_to_python_windows(self, microseconds):
         return ZERO_EPOCH + timedelta(seconds=microseconds)
 
-    def TIMESTAMP_LTZ_to_python(self, microseconds):
+    def TIMESTAMP_LTZ_to_python_one_field(self, microseconds):
         tzinfo = self._get_session_tz()
         return datetime.fromtimestamp(microseconds, tz=tzinfo)
 
-    def TIMESTAMP_LTZ_to_python_windows(self, microseconds):
+    def TIMESTAMP_LTZ_to_python_two_field(self, epoch: float, fraction: float):
         tzinfo = self._get_session_tz()
+        return datetime.fromtimestamp(epoch, tz=tzinfo) + timedelta(seconds=fraction)
+
+    def TIMESTAMP_LTZ_to_python_one_field_windows(self, microseconds):
```

Comment:
Add type-hints and maybe a doc-string that describes why this needs to act differently on Windows